src/components/ProfilePage/elements/VisibilitySelect.messages.jsx

```diff
@@ -0,0 +1,46 @@
+import { defineMessages } from 'react-intl';
+
+const messages = defineMessages({
```

Comment:
Would it make sense to just use the FormControls messages file with a note in the code that they're the same?